package org.json;

import java.util.List;
import kotlin.Metadata;
import kotlin.jvm.internal.Intrinsics;

/* JADX INFO: loaded from: classes3.dex */
@Metadata(d1 = {"\u0000(\n\u0002\u0018\u0002\n\u0002\u0010\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0010\u000e\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0010\u0002\n\u0000\n\u0002\u0010 \n\u0002\b\u0003\bf\u0018\u00002\u00020\u0001J\"\u0010\t\u001a\u00020\b2\u0006\u0010\u0003\u001a\u00020\u00022\b\u0010\u0005\u001a\u0004\u0018\u00010\u00042\u0006\u0010\u0007\u001a\u00020\u0006H\u0016J\u001e\u0010\t\u001a\u00020\b2\f\u0010\u000b\u001a\b\u0012\u0004\u0012\u00020\u00020\n2\u0006\u0010\f\u001a\u00020\u0002H\u0016ø\u0001\u0000\u0082\u0002\u0006\n\u0004\b!0\u0001¨\u0006\rÀ\u0006\u0001"}, d2 = {"Lcom/ironsource/tn;", "", "Lcom/ironsource/y;", j5.p, "", oo.d, "Lcom/ironsource/nj;", "publisherDataHolder", "", "a", "", "waterfallInstances", "winnerInstance", "mediationsdk_release"}, k = 1, mv = {1, 8, 0})
public interface tn {

    /* JADX INFO: renamed from: com.ironsource.tn$-CC, reason: invalid class name */
    public final /* synthetic */ class CC {
        public static void $default$a(tn _this, y instance, String str, nj publisherDataHolder) {
            Intrinsics.checkNotNullParameter(instance, "instance");
            Intrinsics.checkNotNullParameter(publisherDataHolder, "publisherDataHolder");
        }

        public static void $default$a(tn _this, List waterfallInstances, y winnerInstance) {
            Intrinsics.checkNotNullParameter(waterfallInstances, "waterfallInstances");
            Intrinsics.checkNotNullParameter(winnerInstance, "winnerInstance");
        }
    }

    void a(y instance, String placementName, nj publisherDataHolder);

    void a(List<? extends y> waterfallInstances, y winnerInstance);
}
